### PR TITLE
Adding Appendix, moving gitlab instructions to appendix, and removing…

### PR DESCRIPTION
**Pull request by logandonley**
… cluster urls

### CODE REVIEW

File: exercises/4-an-enslaved-hope/README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ git push
 2. On the job's configure page; set the Branch Sources to `git`
 ![multibranch-select-git](../images/exercise4/multibranch-select-git.png)
 
-2. Fill in the Git settings with your `todolist-api` GitLab url and set the credentials as you've done before. `https://gitlab.apps.lader.rht-labs.com/<YOUR_NAME>/todolist-api.git`
+2. Fill in the Git settings with your `todolist-api` GitLab url and set the credentials as you've done before. `https://gitlab.apps.<OpenShift domain><YOUR_NAME>/todolist-api.git`
```
Comment (oybed) [inline exercises/4-an-enslaved-hope/README.md]: need the `/` before `<YOUR_NAME>`

File: exercises/4-an-enslaved-hope/README.md [hunk 2]
```diff
@@ -324,7 +324,7 @@ ansible-playbook apply.yml -e target=tools \
      -e "filter_tags=zap"
 ```
 
-3. Head to https://console.lader.rht-labs.com on OpenShift and move to your ci-cd project > builds. You should see `jenkins-slave-zap` has been built.
+3. Head to <OpenShift domain> on OpenShift and move to your ci-cd project > builds. You should see `jenkins-slave-zap` has been built.
```
Comment (oybed) [inline exercises/4-an-enslaved-hope/README.md]: this should be the "console" reference - i.e.: not the "domain"

File: exercises/4-an-enslaved-hope/README.md [hunk 3]
```diff
@@ -347,7 +347,7 @@ git checkout exercise4/zap-and-arachni params/jenkins-slave-arachni docker/jenki
 
 3. Update the `jenkins-slave-arachni` files `SOURCE_REPOSITORY_URL` to point to your GitLab's hosted version of the `enablement-ci-cd` repo.
 ```
-SOURCE_REPOSITORY_URL=https://gitlab.apps.lader.rht-labs.com/<GIT_USERNAME>/enablement-ci-cd.git
+SOURCE_REPOSITORY_URL=https://gitlab.apps.<OpenShift domain><GIT_USERNAME>/enablement-ci-cd.git
```
Comment (oybed) [inline exercises/4-an-enslaved-hope/README.md]: Need the `/`

File: exercises/4-an-enslaved-hope/README.md [hunk 4]
```diff
@@ -372,7 +372,7 @@ ansible-playbook apply.yml -e target=tools \
      -e "filter_tags=arachni"
 ```
 
-3. Head to https://console.lader.rht-labs.com on OpenShift and move to your ci-cd project > builds. You should see  `jenkins-slave-arachni`.
+3. Head to <OpenShift domain> on OpenShift and move to your ci-cd project > builds. You should see  `jenkins-slave-arachni`.
```
Comment (oybed) [inline exercises/4-an-enslaved-hope/README.md]: `s/domain/console`

**Comment by springdo** [comment]
@logandonley, @oybed  - why is this change needed?

In previous PR, we removed the `<DOMAIN_NAME>` from all the Openshift URL. 
We had overwhelming feedback  from the first run of the course saying it was deeply confusing and tedious /unnecessary in the lab flow to have to keep editing the URL as part of the instructions. 

As for the GitLab change.... I understand why you're doing this, but how many leaners were unable to skip the section and proceed as instructed by the tutor? If we do move it to an appendix, you will also need to update the later lesson where we add WebHook for git too to the same appendix.

**Comment by oybed** [comment]
@springdo this is a "feature branch" that there will be multiple PRs for - it's in preparation for the session next week. If you prefer to have this done in a fork rather than here, we can move it. Once we have a feature branch that we believe is good to merge into the "main content", we'll submit a PR for that and we can discuss from an overall perspective. 

Basically, this feature branch will contain changes based on the feedback received for the last NA session and changes we believe will benefit the execution for next week.

**Pull request closed without merging**